src/main/java/core/basesyntax/utils/transaction/handlers/impl/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.utils.transaction.handlers.impl;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.handlers.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void perform(FruitTransaction transaction, StorageImpl storage) {
+        storage.updateEntry(transaction.getProduct(), -transaction.getQuantity());
+    }
```

Comment:
The logic in the `perform` method of `ReturnOperation` seems to be incorrect. According to the task description, a return operation should increase the quantity of fruit in the storage, but here you're subtracting the quantity (`-transaction.getQuantity()`). This will decrease the quantity instead of increasing it.